src/main/scala/Day05.scala

```diff
@@ -0,0 +1,78 @@
+
+
+import scala.annotation.tailrec
+import scala.io.*
+import scala.language.postfixOps
+
+object Day05 extends App:
+
+  val day: String =
+    this.getClass.getName.drop(3).init
+
+  val start1: Long =
+    System.currentTimeMillis
+
+  var polymer =
+    Source
+      .fromResource(s"input$day.txt")
+      .mkString
+
+  /*
+  Make a list with all possible units
+    Return: List(('a','A'),('A','a'),('b','B'),...
+  */
+  val lowercase = ('a' to 'z').toList
+  val uppercase = ('A' to 'Z').toList
+  val table = lowercase.zip(uppercase).flatMap{
+    case (i, j) => Seq(s"$i$j", s"$j$i")
+  }
+
+  /*
+  Replace string s with replacement string for all strings in List t
+   */
+  def replaceStrings(s: String, t: List[String], replacement: String): String = {
+    t.foldLeft(s)((curLine, str) => curLine.replace(str, replacement))
+  }
+
+  /*
+  Apply function f on s and continue while the result of f(s) is shorter then the length of s
+  */
+  def destroy_units(s: String)(f: (String, List[String], String) => String, t: List[String], r: String): String = {
```

Comment:
minor syntax - prefer `lowerCamelCase` for terms, and `UpperCamelCase` for types